index.scss

```diff
@@ -1,13 +1,22 @@
+@import "@nib-styles/colors";
+@import "@nib-styles/sass-spacing";
+
 .micro-header {
-  padding: 9px;
+  @include py(2);
+  position: relative;
   text-align: center;
-  background-color: hsl(134, 100%, 29%);
+}
+
+.micro-header--green {
+  background-color: $color-queen-elizabeth;
+}
+
+.micro-header--white {
+  background-color: #fff;
```

Comment:
We're probably never going to change white to another color, but we do have `$color-white`